src/Extend/View.php

```diff
@@ -41,12 +43,58 @@ public function namespace(string $namespace, $hints): self
         return $this;
     }
 
+    /**
+     * Override an existing namespace of Laravel views.
+     *
+     * **If you're only replacing one or a few views, you probably want to `->prepend()` the new views instead.**
+     *
+     * Views are PHP files that use the Laravel Blade syntax for creation of server-side generated HTML.
+     * Flarum core uses them for error pages, the installer, HTML emails, and the skeletons for the forum and admin sites.
+     * To replace views, you will need to put them in a folder in your extension, and register that folder under an existing namespace.
```

Comment:
Let's factor out this explanation to the extender's docblock. Too much repetition IMO.